docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+
+services:
+   db:
+     image: sameersbn/mysql
```

Comment:
Why are we not using the Docker supported container for MySQL. Wouldn't that be easier to maintain in the future?